src/components/Layouts/ReportLayout.less

```diff
@@ -112,14 +114,15 @@ h1 {
       &.section-itemsSection,
       .section-list:first-child,
       .section-number,
-      .section-table,
+      .section-table:first-child,
       .section-markdown,
       .section-duration,
       .section-grouped-list,
       .global-section {
         &:not(.page-break-section) {
           border: 1px solid #DADBDB;
           padding: 10px 12px;
+          min-height: 140px;
```

Comment:
from where is this value coming from

Reply:
https://www.figma.com/proto/o1YarvORod1pfGvejrw2hk/objects-reports?page-id=156%3A12956&node-id=156%3A13059&viewport=643%2C588%2C0.08749071508646011&scaling=min-zoom&starting-point-node-id=173%3A13959